source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -180,11 +197,149 @@ void TextureBaker::writeBakedDocument(NodePtr shader, const FilePath& filename)
     writeToXmlFile(bakedTextureDoc, filename);
 }
 
-FilePath TextureBaker::generateTextureFilename(OutputPtr output)
+// Helper function to generate mtlx filename
```

Comment:
For consistency, I would keep the high-level descriptions of these functions in the header, so that they're visible to clients without access to the source.